### ISSUE TITLE
[Config Support]: RTMP Very Slow

### ISSUE THREAD

**Issue by setterlee** [blakeblackshear/frigate#5888]
### Describe the problem you are having

Hi, I'm starting to use go2rtc to add a Chinese generic camera to Frigate. Originally, this camera was throwing errors when I tried to configure it with the original RTSP URL. However, after adding this camera to go2rtc and using the URL provided by go2rtc, it works very well.

After that, I noticed that using the custom WebRTC Lovelace card for this camera looks very fluid compared to my Picture card and the cameras as Home Assistant entities. So, I decided to activate the RTMP on Frigate to add these URLs to go2rtc and test the custom WebRTC Lovelace card with previous cameras through RTMP. However, I noticed that the RTMP streaming of my previous cameras looks very slow.

- Could my camera setup be making the RTMP slow?
- I noticed that RTMP is deprecated in the preview version. Should I wait for the stable version to solve this issue?
- The preview version has a new go2rtc section in the config file. Will this version include go2rtc out of the box? Because I just installed it on my server with a Docker Compose file.

### Version

0.11.1-2EADA21

### Frigate config file

```yaml
mqtt:
  host: 127.0.0.1
  user: XXXXX
  password: XXXXXX
detectors:
  coral:
    type: edgetpu
    device: usb
cameras:
  camera_h264:
    mqtt:
      timestamp: False
      bounding_box: False
      crop: True
      quality: 100
      height: 500
    ffmpeg:
      inputs:
        - path: rtsp://XXXX:XXXX@192.168.XXX.XXX:554/stream1
          roles:
            - detect
            - record
            - rtmp
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
    objects:
      track:
        - person
        - car
        - cat
    detect:
      width: 2560
      height: 1440
      stationary:
        interval: 0
        threshold: 50
        max_frames:
          default: 3000
          objects:
            car: 100
    zones:
      calle:
        coordinates: 0,403,634,316,1344,285,1911,320,2379,385,2401,292,2560,368,2560,0,0,0
        objects:
          - person
          - car
      estacionamiento_delantero:
        coordinates: 0,1440,1989,1440,1201,484,806,515,390,565,0,609
        objects:
          - person
          - cat
      frontis_izquierdo:
        coordinates: 1406,460,1712,457,2094,455,2560,745,2457,955,2560,1042,2560,1440,1966,1440,1121,480
        objects:
          - person
          - cat
      frontis_derecho:
        coordinates: 2350,457,2497,536,2451,683,2076,453
        objects:
          - person
          - cat
      muro_perimetral_delantero_derecho:
        coordinates: 2411,288,2560,366,2489,547,2350,460
        objects:
          - person
          - cat
    record:
      events:
        required_zones:
          - calle
          - estacionamiento_delantero
          - frontis_izquierdo
          - frontis_derecho
          - muro_perimetral_delantero_derecho
        retain:
          objects:
            car: 5
    snapshots:
      height: 500
      crop: False
      required_zones:
        - calle
        - estacionamiento_delantero
        - frontis_izquierdo
        - frontis_derecho
        - muro_perimetral_delantero_derecho
    rtmp:
      enabled: True
  camera_chinese_generic:
    mqtt:
      timestamp: False
      bounding_box: False
      crop: True
      quality: 100
      height: 500
    ffmpeg:
      inputs:
        - path: rtsp://192.168.XXX.XXX:8554/camera_lavandero
          roles:
            - detect
            - record
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
    objects:
      track:
        - person
      filters:
        person:
          min_area: 100000
    snapshots:
      height: 500
      crop: False
    
 
 
rtmp:
  enabled: False
record:
  enabled: True
  retain:
    days: 14
    mode: motion
  events:
    retain:
      default: 14
      mode: active_objects
snapshots:
  enabled: True
  retain:
    default: 14
birdseye:
  enabled: True
  mode: motion
  width: 1280
  height: 720
logger:
  # Optional: default log level (default: shown below)
  default: info
  # Optional: module by module log level configuration
  logs:
    frigate.mqtt: error
ffmpeg:
  hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format yuv420p
```

### Relevant log output

```shell
frigate    | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
frigate    | [s6-init] ensuring user provided files have correct perms...exited 0.
frigate    | [fix-attrs.d] applying ownership & permissions fixes...
frigate    | [fix-attrs.d] done.
frigate    | [cont-init.d] executing container initialization scripts...
frigate    | [cont-init.d] done.
frigate    | [services.d] starting services
frigate    | [services.d] done.
frigate    | [2023-03-31 21:11:52] frigate.app                    INFO    : Starting Frigate (0.11.1-2eada21)
frigate    | Starting migrations
frigate    | [2023-03-31 21:11:52] peewee_migrate                 INFO    : Starting migrations
frigate    | There is nothing to migrate
frigate    | [2023-03-31 21:11:52] peewee_migrate                 INFO    : There is nothing to migrate
frigate    | [2023-03-31 21:11:53] frigate.app                    INFO    : Output process started: 218
frigate    | [2023-03-31 21:11:53] ws4py                          INFO    : Using epoll
frigate    | [2023-03-31 21:11:53] frigate.app                    INFO    : Camera processor started for camera_h264: 225
frigate    | [2023-03-31 21:11:53] frigate.app                    INFO    : Camera processor started for camera_chinese_generic: 233
frigate    | [2023-03-31 21:11:53] frigate.app                    INFO    : Capture process started for camera_h264: 235
frigate    | [2023-03-31 21:11:53] frigate.app                    INFO    : Capture process started for camera_chinese_generic: 258
frigate    | [2023-03-31 21:11:53] ws4py                          INFO    : Using epoll
frigate    | [2023-03-31 21:11:53] detector.coral                 INFO    : Starting detection process: 216
frigate    | [2023-03-31 21:11:53] frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate    | [2023-03-31 21:11:55] frigate.edgetpu                INFO    : TPU found
```

### Frigate stats

```json
{
    "camera_h264": {
        "camera_fps": 5.0,
        "capture_pid": 235,
        "detection_fps": 6.0,
        "pid": 225,
        "process_fps": 5.1,
        "skipped_fps": 0.0
    },
    "camera_chinese_generic": {
        "camera_fps": 5.0,
        "capture_pid": 258,
        "detection_fps": 3.1,
        "pid": 233,
        "process_fps": 5.0,
        "skipped_fps": 0.0
    },
    "detection_fps": 9.5,
    "detectors": {
        "coral": {
            "detection_start": 0.0,
            "inference_speed": 12.85,
            "pid": 216
        }
    },
    "service": {
        "latest_version": "0.11.1",
        "storage": {
            "/dev/shm": {
                "free": 865.7,
                "mount_type": "tmpfs",
                "total": 897.6,
                "used": 31.8
            },
            "/media/frigate/clips": {
                "free": 146902.8,
                "mount_type": "ext4",
                "total": 489997.2,
                "used": 318128.7
            },
            "/media/frigate/recordings": {
                "free": 146902.8,
                "mount_type": "ext4",
                "total": 489997.2,
                "used": 318128.7
            },
            "/tmp/cache": {
                "free": 992.6,
                "mount_type": "tmpfs",
                "total": 1000.0,
                "used": 7.4
            }
        },
        "temperatures": {},
        "uptime": 16,
        "version": "0.11.1-2eada21"
    }
}
```

### Operating system

Other

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment]
RTMP being slow is not a surprise, one of the many reasons it's being deprecated. 

And yes, go2rtc is going to be included in frigate 0.12

**Comment by setterlee** [comment]
Well, I will start to play with the 0.12 right now… thanks Nick
